css/style.css

```diff
@@ -0,0 +1,658 @@
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  color: #333;
+}
+
+.image-container {
+  margin-bottom: 30px;
+}
+
+.image-container img {
+  width: 100%;
+  height: auto;
+}
+
+.image-container div {
+  width: 80%;
+}
+
+.main-section-footer-text p {
+  color: rgb(102, 102, 102);
+  font-size: 0.75rem;
+  line-height: 1rem;
+}
+
+.bottom-navbar-offer p {
+  font-size: 16px;
+  text-decoration: none;
+  color: whitesmoke;
+  cursor: pointer;
+}
+
+.subscribe div p {
+  text-align: center;
+  margin: 5px;
+  font-size: 16px;
+}
+
+.img-text div a {
+  color: rgb(51, 51, 51);
+  font-size: 14px;
+  line-height: 17px;
+  text-decoration: none;
+  font-weight: 500;
+  padding-bottom: 0;
+  border-bottom: none;
+  margin-left: 5px;
+  align-self: center;
+}
+
+.bottom-navbar-text p a {
+  font-size: 16px;
+  text-decoration: none;
+  color: whitesmoke;
+}
+
+.main-section-footer-text p a {
+  color: rgb(102, 102, 102);
+  text-decoration: underline;
+}
+
+.subscribe div p a {
+  color: rgb(50, 104, 145);
+  text-decoration: none;
+  font-size: 16px;
+}
+
+.image-container div p {
+  color: #666;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  text-align: left;
+}
+
+.data-icons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.data-icons span p {
+  font-weight: 500;
+  color: #333;
+  font-size: 0.8125rem;
+  line-height: 1rem;
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.social-media-buttons {
+  height: 28px;
+  width: 28px;
+  border-radius: 50%;
+  border: 0;
+  margin-right: 10px;
+  cursor: pointer;
+}
+
+.social-media-buttons:hover {
+  background-color: white;
+  border: 1px whitesmoke solid;
+}
+
+.suscribe {
+  align-self: center;
+  color: #333;
+}
+
+.subscribe-container-top-border {
+  border: 1px solid rgb(204, 204, 204);
+  width: 72px;
+  margin: 0 auto 18px;
+}
+
+.subscribe-container-bottom-border {
+  border: 1px solid rgb(204, 204, 204);
+  width: 72px;
+  margin: 18px auto 0;
+}
+
+.aside {
+  position: absolute;
+  top: 2000px;
+  width: 230px;
+  right: 10%;
+}
+
+.aside h2 {
+  color: black;
+  font-weight: 700;
+  margin-top: 0;
+  margin-bottom: 15px;
+  padding-bottom: 12px;
+  font-size: 14px;
+  line-height: 14px;
+  letter-spacing: 0.02em;
+  border-bottom: 1px solid rgb(226, 226, 226);
+}
+
+.img-text {
+  padding-right: 20px;
+}
+
+.img-text div {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.telescope-image-container img {
+  height: auto;
+  width: 800px;
+}
+
+.footer-tittle img {
+  margin-right: 60px;
+}
+
+.img-text div img {
+  width: 70px;
+  height: 70px;
+}
+
+.column-grid li p a {
+  font-size: 14px;
+  text-decoration: none;
+  color: rgb(0, 0, 0);
+}
+
+.ul-text-1 li p a {
+  font-size: 12px;
+  text-size-adjust: 100%;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.ul-text-2 li p a,
+.ul-text-3 li p a {
+  font-size: 11.5px;
+  text-size-adjust: 100%;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  You might directl going to `.ul-text-2 a { }` delete other unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_